pkg/cmd/pr/create/create.go

```diff
@@ -696,6 +704,73 @@ func submitPR(opts CreateOptions, ctx CreateContext, state shared.IssueMetadataS
 		return err
 	}
 
+	if opts.DryRun {
+		out := opts.IO.Out
+		if opts.IO.IsStdoutTTY() {
+			fmt.Fprint(out, "Would have created a Pull Request with:\n")
+			fmt.Fprintf(out, "title:\t%s\n", params["title"])
+			fmt.Fprintf(out, "draft:\t%t\n", params["draft"])
+			if len(state.Labels) != 0 {
+				fmt.Fprintf(out, "labels:\t%v\n", state.Labels)
+			}
+			if len(state.Reviewers) != 0 {
+				fmt.Fprintf(out, "reviewers:\t%v\n", state.Reviewers)
+			}
+			if len(state.Assignees) != 0 {
+				fmt.Fprintf(out, "assignees:\t%v\n", state.Assignees)
+			}
+			if len(state.Milestones) != 0 {
+				fmt.Fprintf(out, "milestones:\t%v\n", state.Milestones)
+			}
+			if len(state.Projects) != 0 {
+				fmt.Fprintf(out, "projects:\t%v\n", state.Projects)
+			}
+			if len(params["body"].(string)) != 0 {
+				fmt.Fprintln(out, "--")
+				fmt.Fprintln(out, params["body"])
+			}
+		} else {
+			cs := opts.IO.ColorScheme()
+			fmt.Fprintf(out, "%s\n", cs.Bold(params["title"].(string)))
+			if len(state.Labels) != 0 {
+				fmt.Fprint(out, cs.Bold("Labels: "))
+				fmt.Fprintln(out, state.Labels)
+			}
+			if len(state.Reviewers) != 0 {
+				fmt.Fprint(out, cs.Bold("Reviewers: "))
+				fmt.Fprintln(out, state.Reviewers)
+			}
+			if len(state.Assignees) != 0 {
+				fmt.Fprint(out, cs.Bold("Assignees: "))
+				fmt.Fprintln(out, state.Assignees)
+			}
+			if len(state.Milestones) != 0 {
+				fmt.Fprint(out, cs.Bold("Milestones: "))
+				fmt.Fprintln(out, state.Milestones)
+			}
+			if len(state.Projects) != 0 {
+				fmt.Fprint(out, cs.Bold("Projects: "))
+				fmt.Fprintln(out, state.Projects)
+			}
```

Comment:
Would this be easier to read and managed to combine the lines?

```suggestion
			if len(state.Labels) != 0 {
				fmt.Fprintf(out, "%s: %s\n", cs.Bold("Labels"), state.Labels)
			}
			if len(state.Reviewers) != 0 {
				fmt.Fprintf(out, "%s: %s\n", cs.Bold("Reviewers"), state.Reviewers)
			}
			if len(state.Assignees) != 0 {
				fmt.Fprintf(out, "%s: %s\n", cs.Bold("Assignees"), state.Assignees)
			}
			if len(state.Milestones) != 0 {
				fmt.Fprintf(out, "%s: %s\n", cs.Bold("Milestones"), state.Milestones)
			}
			if len(state.Projects) != 0 {
				fmt.Fprintf(out, "%s: %s\n", cs.Bold("Projects"), state.Projects)
			}
```

Reply:
Thanks for this. I like that approach, just for the record I used the same pattern in https://github.com/cli/cli/blob/385bb48f210bb90c80fc16053999f09f913283c7/pkg/cmd/pr/view/view.go#L213-L233